asteroidsyncservice_config.h.in

```diff
@@ -21,7 +21,7 @@
 #define CONFIG_H
 
 #define VERSION "${asteroidsyncservice_VERSION_MAJOR}.${asteroidsyncservice_VERSION_MINOR}"
-#cmakedefine UBUNTU_TOUCH_PLATFORM
-#cmakedefine SAILFISHOS_PLATFORM
+#cmakedefine01 UBUNTU_TOUCH_PLATFORM
+#cmakedefine01 SAILFISHOS_PLATFORM
```

Comment:
Why is that necesssary ?

Reply:
In the places that these were tested in the code they used #ifdef ... #elif, but that logic was faulty because the first clause tests for definition and the second for value.  This change defines both with either a 1 or 0, making it easier to use with #if .. #elif.